### DISCUSSION TITLE
Plausible is only counting 10% of my visits/pages

### DISCUSSION THREAD

**Discussion by marcbria** [plausible/analytics#2679] [Self-Hosted Support] [answered]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

I set up a plausible installation based on docker images and all worked fast and smothly.
Then I add the plausible script in my header to count my subdomain and also the global domain as follows:
```
<script defer data-domain="foo.org/folder,foo.org" src="https://myserver.org/js/script.js"></script>
```

A week after I compare plausible statistics (visitors, pages...) with my own CMS statistics or Google Analytics and plausible is only counting around 10% of the visits.

Is there a way to track what is happening?

PD: I would also like to place your script in footer (as is a good practice) but I'm not sure if this will have any impact in metrics.

### Expected behavior

I expect plausible count same visits (or very close) to the ones I get from google's or my CMS.

### Environment

```markdown
- OS: Debian + Docker
- Browser: Not relevant
- Browser Version: Not relevant
```

**Comment by Tony-MF** [comment]
Can you post here a link to your site where the tracking code is deployed? I can have a look.

**Reply by marcbria** [reply]
Sure. 

Take this one as an example: https://atheneadigital.net
PDFs are not tracked yet (I need to implement a plugin to change it's header) but regular pages are supposed to work fine.

BTW, do you mind to confirm if it's important to add the script in the header or if this could be added in footer?
In my case is more important to offer the best user experience than monitoring.

Thanks for your help,
m.

**Comment by marcbria** [comment, marked as answer]
Hi,

Long story short... I was mistaken.

As far as I was worried about not counting visits in the right way... I made a comparative between 4 different tools (plausible, matomo, analytics, and our "cms") to discover what was wrong, and I found the answer was as simple as I was comparing different things (visitors vs pageviews) and analytics was counting wrongly.

Here you have the comparative, that is not very accurate, but good enough for my peace of mind:
https://forum.pkp.sfu.ca/t/comparison-of-the-results-in-the-statistical-tools-of-the-journals/78119

Thanks a lot for this wonderful piece of free software,
m.
